network_simulator/link.py

```diff
@@ -17,8 +17,8 @@ class Link(object):
     """
 
     def __init__(self, end_1_addr=None, end_2_addr=None, end_1_device=None,
-                 end_2_device=None, buffer=None, static_delay_sec=None,
-                 capacity_bps=None):
+                 end_2_device=None, buffer=None, buffer_occupancy=0,
```

Comment:
The changes to link.py should wait until the PR #4 is merged in, since that makes a lot of substantial changes to link.py and this constructor.

With the new link.py, you will be able to find the buffer_occupancy by just finding the qsize() of the LinkBuffer's queue (qsize() is fine to use since we don't have multi-threading happening). Having an additional variable like this is not a good idea.